### ISSUE TITLE
Managing Google Accounts/Forms

### ISSUE THREAD

**Issue by dgaylordCS4All** [nycdoe-cs4all/blueprint-learning-site#91]
How to manage student Google Accounts and how to create Google Forms for use as assessments

**Comment by aankit** [comment]
@dgaylordCS4All I'm not sure what this means - can you rephrase as what you expect and and what is currently happening?

**Comment by dgaylordCS4All** [comment]
From what I remember, these were from a teacher who wanted more resources on the blueprint.

First was for how to manage Google Accounts, specifically as it pertains to CS Instruction. I would say this is not a CS thing though, and I wanted it here to relay what some teachers are asking for, but I believe this is a DIIT thing.

As for Google forms, they wanted more resources on how they can be used for assessments. Again, this is not something specific for CS, so these resources might already exist somewhere.

Looking back at these two things, I don't think they are needed on the blueprint. They don't deal with anything specific to CS.
